src/main/java/org/terasology/signalling/componentSystem/SignalSystem.java

```diff
@@ -108,6 +108,9 @@ public void shutdown() {
         signalNetwork = null;
     }
 
+    /**
+     * This method ensures that computers cannot be faster than the processing interval
+     */
     @Override
```

Comment:
So @pollend , this one is for you.

I'm of the opinion that javadoc is the high-level description of what a function does. update(), as the name implies, updates the object every frame. If anything, this object should inherit this javadoc from it's superclass.
The `ensure that computers cannot be faster than processing interval` part should, IMO, be a normal comment inside the update method to explain succinctly the if condition, and not a part of the javadoc.

Your thoughts?

Reply:
I agree with @vampcat.